microprofile.jdt/com.redhat.microprofile.jdt.core/src/main/java/com/redhat/microprofile/jdt/internal/core/java/ChangeUtil.java

```diff
@@ -0,0 +1,151 @@
+package com.redhat.microprofile.jdt.internal.core.java;

```

Comment:
Do we need a header in this file? Is this file originally from org.eclipse.jdt.ls.core.internal?

Reply:
When you did a copy/paste file, you need to keep the original hearder.